src/index.js

```diff
@@ -57,12 +59,14 @@ const Client = function Client(config = {}) {
   };
 
   this.utils = {
-    groupTraits: trait.group,
+    // @deprecated - let's remove the `groupTraits` in favor of `traits.group` convention
+    groupTraits: traitsUtils.group,
```

Comment:
Yes, that was the idea not to remove it suddenly just in case some customer may be using it.